lib/esbonio/esbonio/lsp/__init__.py

```diff
@@ -162,6 +163,14 @@ def on_open(ls: RstLanguageServer, params: DidOpenTextDocumentParams):
     def on_change(ls: RstLanguageServer, params: DidChangeTextDocumentParams):
         pass
 
+    @server.command(ESBONIO_SERVER_SPHINX_BUILD_UNSAVED_FILE)
+    def trigger_build_for_usaved_file(
```

Comment:
If this can be turned into just a trigger for the `build()` method (see comments below), we can add a no-op `build()` method to the `RstLanguageServer` which should fix some of the `mypy` errors

Reply:
We could rename it and allow a set of optional parameters (unsaved file paths) and move the priming (of the callbacks) into the build function.